.github/workflows/continuous-delivery.yaml

```diff
@@ -39,6 +39,16 @@ jobs:
           commit-assets: |
             ./dist
           node-module: true
+          release-rules: |
+            [
+              { "release": "patch", "type": "build" },
+              { "release": "patch", "type": "chore", "scope": "deps" },
+              { "release": "patch", "type": "chore", "scope": "deps-dev", "message": "bump typescript from*" },
+              { "release": "patch", "type": "chore" },
```

Comment:
I don't see how would that exclude `deps-dev` from being released. Did you test it or is there some documentation that would explain such behavior? I don't see how would that work.

Reply:
Changed, this should do the trick. I think!

There are no explicit docs around how release rules work (is order important?), only examples to paint some very specific use cases.